_data/s23deepak/s23deepak.md

```diff
@@ -0,0 +1,7 @@
+---
+name : Deepak Swaminathan
+institution : Gandhi Institute of Technology and Management, Visakhapatnam
+profile_pic : s23deepak.png

```

Comment:
Thanks for the changes - however, the file extension is incorrect (your image is a ```.jpg``, not a ```.png```).

Reply:
Thanks for the suggestions, I have changed them.